web/templates/user_profile_modal.hbs

```diff
@@ -18,16 +18,18 @@
                         <i class="zulip-icon zulip-icon-bot" aria-hidden="true"></i>
                     {{/if}}
                     <span class="user-profile-name">{{> user_full_name name=full_name}}</span>
-                    <a id="user_profile_manage_own_copy_link" class="copy-link-to-user-profile" tabindex="0" data-user-id="{{user_id}}">
-                        <i class="fa fa-link tippy-zulip-tooltip user-profile-manage-own-copy-link-button" data-tippy-content="{{t 'Copy link to profile' }}" aria-hidden="true"></i>
+                    <a class="copy-link-to-user-profile user-profile-manage-own-copy-link-button" tabindex="0" data-user-id="{{user_id}}">
+                        <i class="zulip-icon zulip-icon-link tippy-zulip-tooltip " data-tippy-content="{{t 'Copy link to profile' }}" aria-hidden="true"></i>
                     </a>
                     {{#if is_me}}
-                        <a id="user_profile_manage_own_edit_link" href="/#settings/profile">
-                            <i class="fa fa-edit tippy-zulip-tooltip user-profile-manage-own-edit-button" data-tippy-content="{{t 'Edit profile' }}" aria-hidden="true"></i>
+                        <a class="user-profile-manage-own-edit-button" href="/#settings/profile">
```

Comment:
I think some of these hunks belong in the previous commit in this PR that renamed things?

Reply:
Ahh, no, it's just that `user_profile_manage_own_edit_link` was not renamed.